shell/platform/linux/fl_accessible_node.h

```diff
@@ -5,17 +5,14 @@
 #ifndef FLUTTER_SHELL_PLATFORM_LINUX_FL_ACCESSIBLE_NODE_H_
 #define FLUTTER_SHELL_PLATFORM_LINUX_FL_ACCESSIBLE_NODE_H_
 
-#include <gtk/gtk.h>
+#include <gio/gio.h>
+#include <atk/atk.h>
 
 #include "flutter/shell/platform/embedder/embedder.h"
 #include "flutter/shell/platform/linux/public/flutter_linux/fl_engine.h"
 
 G_BEGIN_DECLS
 
-// ATK doesn't have the g_autoptr macros, so add them manually.
-// https://gitlab.gnome.org/GNOME/atk/-/issues/10
-G_DEFINE_AUTOPTR_CLEANUP_FUNC(AtkObject, g_object_unref)
-
 #define FL_TYPE_ACCESSIBLE_NODE fl_accessible_node_get_type()
```

Comment:
```suggestion
// ATK g_autoptr macros weren't added until 2.37. Add them manually.
// https://gitlab.gnome.org/GNOME/atk/-/issues/10
#if !ATK_CHECK_VERSION(2, 37, 0)
G_DEFINE_AUTOPTR_CLEANUP_FUNC(AtkObject, g_object_unref)
#endif

#define FL_TYPE_ACCESSIBLE_NODE fl_accessible_node_get_type()
```

Reply:
Hrm looks like adding the suggested commit is blocked for this branch, but you may be able to apply it.